app/upgrades/v5/upgrades.go

```diff
@@ -58,11 +61,15 @@ func CreateUpgradeHandler(
 			bk.SetDenomMetaData(ctx, TestnetDenomMetadata)
 		}
 
+		if types.IsMainnet(ctx.ChainID()) {
+			ResolveAirdrop(ctx, ck)
+		}
+
 		// define from versions of the modules that have a new consensus version
 
 		// migrate fee market module, other modules are left as-is to
 		// avoid running InitGenesis.
-		vm[feemarkettypes.ModuleName] = 2
+		vm[feemarkettypes.ModuleName] = 3
```

Comment:
I think the `RunMigrations` expects the `fromVersion` , and `feemarket` is going from consensus version 2 to 3, right? So shouldn’t it be `2` in the migration?

Reply:
correct, this should be `2`
```suggestion
		vm[feemarkettypes.ModuleName] = 2
```